### ISSUE TITLE
Absolute file path linting rule does not detect absolute working directories

### ISSUE THREAD

**Issue by OliverGerstl** [flowr-analysis/flowr#2611]
The absolute file paths linting rule does not detect absolute file paths in setter functions for working directories.

For example, the query for `setwd("C:/Users/hamiltsa/Desktop")`
```
:query @linter rules:absolute-file-paths "setwd('C:/Users/hamiltsa/Desktop')"
```
returns no result
```
Query: linter (2 ms)
   ╰ **Absolute Paths** (absolute-file-paths):
       ╰ _Metadata_: <code>totalConsidered: 1, totalUnknown: 0, searchTimeMs: 2, processTimeMs: 0</code>
All queries together required ≈2 ms (1ms accuracy, total 3 ms)
```
